source/develop/developer-guide/vdsm/developers.html.md

```diff
@@ -36,18 +36,22 @@ VDSM requires Python 2 as your /usr/bin/python.
 
 Install the following packages before you attempt to build VDSM:
 
-       yum install make autoconf automake pyflakes logrotate python-pep8 libvirt-python python-devel \
+       PYTHON3=python3
+       YUM=dnf
+       $YUM install make autoconf automake pyflakes logrotate python-pep8 libvirt-python python-devel \
        python-nose rpm-build sanlock-python genisoimage python-odict python-pthreading libselinux-python\
        python-ethtool m2crypto python-dmidecode python-netaddr python-inotify python-argparse git \
        python-cpopen bridge-utils libguestfs-tools-c pyparted openssl libnl3 libtool gettext-devel python-ioprocess \
-       policycoreutils-python python-simplejson python-blivet python-six mom ovirt-vmconsole
+       policycoreutils-python python-simplejson python-blivet python-six mom ovirt-vmconsole \
+       $PYTHON3 $PYTHON3-six $PYTHON3-nose
 
-On EL7.1, pep8 is not available, and the version in pypi is too new, failing the build. So install pip, and then pep8 1.5.6:
+On EL7 you need some modifications to the above lines. PYTHON3 should be set to python34. YUM should be set to 'yum'.
+pep8 is not available, and the version in pypi is too new, failing the build. So install pip, and then pep8 1.5.6:
 
```

Comment:
Also pep8 is available in CentOS Virt SIG: http://cbs.centos.org/koji/buildinfo?buildID=4630 and in epel: http://koji.fedoraproject.org/koji/buildinfo?buildID=618188